routes/service.php

```diff
@@ -54,6 +54,15 @@
         'as' => 'admin.vouchers.retirebatch',
         'uses' => 'Admin\VouchersController@retireBatch',
     ]);
+    Route::get('vouchers/{voucher}', [
```

Comment:
VoucherCotnroller::viewOne() has a parameter of $id, perhaps use {id} here for consitency?

Reply:
This has been changed.